README.md

```diff
@@ -24,6 +24,21 @@ When all is runnning we can start the proxy:
 > * change GARGOYLE_KEYCLOAK_URL in the docker-compose.yml
 > * change the ranger.plugin.s3.policy.rest.url in ranger-s3-security.xml
 
+NOTE: `ranger-s3-security.xml` must exist in resources directory or in directory from classpath
```

Comment:
Maybe add this node just after sbt run command so it's clear this is required. Also add directory currently expected (/etc/gargoyle)